tuf/formats.py

```diff
@@ -996,6 +1020,42 @@ def make_fileinfo(length, hashes, custom=None):
 
 
 
+def make_versioninfo(version_number):
+  """
+  <Purpose>
+    Create a dictionary conformant to 'VERSIONINFO_SCHEMA'.  This dict
+    describes both metadata and target files.
+
+  <Arguments>
+    version_number:
+      An integer representing the version of a particular metadata role.
+      The dictionary returned by this function is expected to be included
+      in Snapshot metadata.
+
+  <Exceptions>
+    tuf.FormatError, if the 'VERSIONINFO_SCHEMA' to be returned
+    does not have the correct format.
+
+  <Side Effects>
```

Comment:
I'm inclined to list None as the side effects. The return value and possible exception are described elsewhere.